### PR TITLE
Add support for selectedBackgroundColor

### LINKED ISSUE #30: Optional selected background color
Thanks for such a useful library!

**Description:**
In some cases users may wish to use a different background color for active segments than the `tintColor`

**Problems I encountered when trying to implement this myself:**
Not sure I did it entirely correctly but I am submitting a PR along with this

Example of using this:

image

### PR DESCRIPTION
**Pull request by jemise111**
Fixes #30

### CODE REVIEW

**Review by yonat — CHANGES_REQUESTED** [review]
Nice, thanks!

Please use the `parent.selectedBackgroundColor` instead of copying it to each segment -- similar to `titleTextAttributes`. (You'll need to remove the `private` from `MultiSelectSegment.updateColors()` so you can call it when the parent's color changes.)

**Comment by jemise111** [comment]
@yonat changes made! Also left a note in the Readme

File: Sources/MultiSelectSegmentedControl.swift [hunk 1]
```diff
@@ -17,6 +17,13 @@ import UIKit
 
     // MARK: - UISegmentedControl Enhancements
 
+    /// Optional selected background color, if different than tintColor
+    @objc public var selectedBackgroundColor: UIColor? {
```
Comment (yonat) [inline Sources/MultiSelectSegmentedControl.swift]: Forgot to mention:

Please make this an `@IBInspectable open dynamic var` To support Interface Builder and UIAppearance.

Thanks!

Comment (yonat) [reply Sources/MultiSelectSegmentedControl.swift]: Also, please move it to the end of the `Appearance` mark.
If you prefer, grant me [edit access to the PR](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) and I'll do these little fixes.)

**Comment by jemise111** [comment]
@yonat done! I think I enabled you to make edits too in case there's anything else

**Comment by yonat** [comment]
Thanks!

**Comment by yonat** [comment]
@CodiumAI-Agent /review

**Pull request merged**